src/angular-vs-repeat.js

```diff
@@ -369,7 +369,7 @@
 
                         repeatContainer.append($beforeContent);
                         repeatContainer.append(childClone);
-                        $compile(childClone)($scope);
+                        $compile(childClone, transclude)($scope);
```

Comment:
Also according to the angular docs, we shouldn't pass the transclude function to compile, but to the returned link function instead

From: https://docs.angularjs.org/api/ng/service/$compile
> Note: Passing a transclude function to the $compile function is deprecated, as it e.g. will not use the right outer scope. Please pass the transclude function as a parentBoundTranscludeFn to the link function instead.

So this line should be:
``` js
$compile(childClone)($scope, null, { parentBoundTranscludeFn: transclude });
```